R/valid_years.R

```diff
@@ -0,0 +1,21 @@
+#' Return available valid years
+#'
+#' @param data_dir character: Path to the root directory
+#'
+#' @return numeric vector of valid years
+#' @export
+
+valid_years <- function(data_dir) {
+  ref <- pipapi::get_aux_table(data_dir, 'interpolated_means')
+  ref <- sort(unique(ref$reporting_year))
+
+  svy <- pipapi::get_aux_table(data_dir, 'survey_means')
+  svy <- sort(unique(svy$reporting_year))
```

Comment:
While reviewing your PR, I realised, that we should probably return both set of survey and interpolated `reporting_years`, since they are not identical.